schemas/migration.md

```diff
@@ -0,0 +1,165 @@
+IPLD Schemas and Migration
```

Comment:
We should pull the heading levels up by one in here, this top one should be `#` or `=====` and the subsequent ones should be `##`.